test/cql-pytest/test_lwt.py

```diff
@@ -27,7 +27,7 @@ def table1(cql, test_keyspace):
 # As discovered in issue #10081, if the row determined by WHERE does NOT
 # exist, Scylla still needs to read the static column, but forgets to do so.
 # this test reproduces this issue.
-def test_lwt_missing_row_with_static(cql, table1):
+def test_lwt_missing_row_with_static(cql, table1, xfail_tablets): # LWT is not supported with tablets yet. See #18066
```

Comment:
Unless I'm missing something, all tests in test_lwt.py require vnodes. The simplest solution would be to just change the definition of table1 to create a table explicitly with vnodes - and all the tests will use it. This hack should be temporary (and refer to #18066) but it can solve you the need to do a dozen changes in this file, and also avoid the lowering test coverage (basically, your patch removes all the LWT tests in the cql-pytest framework).